Kickstarter-iOS/Views/Controllers/CommentsViewControllerTests.swift

```diff
@@ -45,8 +41,11 @@ internal final class CommentsViewControllerTests: TestCase {
   }
 
   func testView_CurrentUser_LoggedOut() {
+    let mockService =
```

Comment:
A lot of these tests were rendering blank because this component of the method was missing.